example_sharedkey_test.go

```diff
@@ -37,22 +39,26 @@ func Example() {
 	// Generate 8 bytes of random salt. Be sure to use crypto/rand for all
 	// of your random number needs
 	salt := make([]byte, 8)
-	rand.Read(salt)
+	if n, err := rand.Read(salt); err != nil {
+		log.Fatal(err)
+	} else if n != 8 {
+		log.Fatal("failed to generate 8 byte salt")
+	}
 
```

Comment:
Again, thank you. Usually I am the one nagging people to check for errors on reading from the RNG.